content/pipeline/getting-started-pipelines.adoc

```diff
@@ -0,0 +1,527 @@
+Open Jenkins in your web browser.
+On the Manage Jenkins page for your installation, navigate to Manage Plugins. 
+Find https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Plugin[Pipeline Plugin] from among the plugins listed on the Available tab. (You can do this by scrolling through the plugin list or by using “Pipeline” as a term to filter results)
+Select the checkbox for Pipeline Plugin.
+Select either *Install without restart* or *Download now and install after restart*. Pipeline plugin installation automatically includes all necessary dependencies. 
+Restart Jenkins.
+
+=== Pipeline Plugin Reference List
+
+The Pipeline plugin works with a suite of related plugins that enhance the pipeline functionality of your Jenkins setup. The additional plugins typically introduce additional pipeline syntax or visualizations.
+
+The table below describes pipeline-related plugins in terms of their importance to pipeline functionality (required, recommended, or optional). To get the basic pipeline functionality, you only need to install the main Pipeline plugin, but recommended plugins add additional capabilities that you will probably want.  
+
+Optional plugins are mainly useful if you are creating pipelines that are related to the technologies that they support. 
+
+
+[options="header"]
+|=======================
+|Plugin Name                     |Description           |Status
+|Pipeline (workflow-aggregator)  | Installs the core pipeline engine and its dependent plugins: 
+Pipeline: API,
+Pipeline: Basic Steps,
+Pipeline: Durable Task Step, 
+Pipeline: Execution Support,
+Pipeline: Global Shared Library for CPS pipeline,
+Pipeline: Groovy CPS Execution,
+Pipeline: Job,
+Pipeline: SCM Step,  
+Pipeline: Step API
+| required
+
+| Pipeline: Stage View
+| Provides a graphical swimlane view of pipeline stage execution (as well as a build history of the stages)
+| recommended
+
+| Multibranch Pipeline
+| Adds "Multibranch Pipeline" item type which allows Jenkins to automatically build branches that contain jenkinsfile
+| recommended
+
+| CloudBees GitHub Branch Source
+| Adds GitHub Organization Folder item type and adds "Github" as a branch source on Multibranch pipelines
+| recommended for teams hosting repositories in Github
+
+| CloudBees Bitbucket Branch Source
+| Adds Bitbucket Team item type and adds "Bitbucket" as a branch source on Multibranch pipelines
+| recommended for teams hosting repositories in Bitbucket
+
+| CloudBees Docker Pipeline
+| Enables pipeline to build and use Docker containers inside pipeline scripts.
+| optional
+
+|=======================
+
+
+Pipeline-related plugins other than those listed above are regularly “whitelisted” as compatible with or designed for Pipeline usage. For more information, see the link:https://github.com/jenkinsci/workflow-plugin/blob/541faf611659e1e6b8f2cbbd3435756b27633db4/COMPATIBILITY.md[Plugin Compatibility With Pipeline] wiki. Current contents of that wiki are summarized in the following table:
```

Comment:
"Plugin Compatibility With Pipeline" is not Wiki. It's just a web page on GitHub